rootfs/api/models/config.py

```diff
@@ -46,7 +46,8 @@ def _migrate_legacy_healthcheck(self):
         success_threshold = int(self.values.get('HEALTHCHECK_SUCCESS_THRESHOLD', 1))
         failure_threshold = int(self.values.get('HEALTHCHECK_FAILURE_THRESHOLD', 3))
 
-        self.healthcheck['livenessProbe'] = {
+        self.healthcheck[''] = {}
```

Comment:
how about `web/cmd` as @slack wan't it to be just for those two proctypes.https://github.com/deis/workflow/pull/424#discussion_r74125645